### ISSUE TITLE
Boolean options are parsed as arguments in CLI

### ISSUE THREAD

**Issue by mondeja** [birjolaxew/svglint#44]
After some debug, I suspect that this is an error with `minimist` dependency of `meow`. If I pass a boolean flag before the input, the input is ignored:

```sh
$ bin/cli.js --config src/rules/examples/elms.js --debug test/svgs/elm.test.svg
---------------------------------- Summary ----------------------------------
- No files linted
$ bin/cli.js --config src/rules/examples/elms.js test/svgs/elm.test.svg --debug
------------------------------------ Log ------------------------------------
[d|lint:test/svgs/elm.test.svg] Started linting
[d|lint:test/svgs/elm.test.svg]   Rules: [ 'valid' ]
[d|rule:valid] Called true
[d|lint:test/svgs/elm.test.svg] Rule finished 'valid'
[d|lint:test/svgs/elm.test.svg] Linting finished with status 'success'
(d) Linting done, 0 to go

---------------------------------- Summary ----------------------------------
✓ 1 valid files.
```

It seems that the boolean options are being treated as arguments:

```sh
$ bin/cli.js --config src/rules/examples/elms.js --debug test/svgs/elm.test.svg test/svgs/elm.test.svg
------------------------------------ Log ------------------------------------
[d|lint:test/svgs/elm.test.svg] Started linting
[d|lint:test/svgs/elm.test.svg]   Rules: [ 'valid' ]
[d|rule:valid] Called true
[d|lint:test/svgs/elm.test.svg] Rule finished 'valid'
[d|lint:test/svgs/elm.test.svg] Linting finished with status 'success'
(d) Linting done, 0 to go

---------------------------------- Summary ----------------------------------
✓ 1 valid files.
```

**Comment by ericcornelissen** [comment]
Thanks for the bug report @mondeja, from my testing it seems the update of #41 will fix this bug. I'll leave this open issue until that is merged and released.

**Comment by mondeja** [comment]
I can confirm that this has been fixed by #41. Thanks @ericcornelissen!
